libs/eo/origin-of-energy/shell/README.md

```diff
@@ -0,0 +1 @@
+# Energy Origin Origin of Energy shell
```

Comment:
suggestion: Consider removing the *Energy Origin* prefix since this is something very specific to Energy Origin and thus no ambiguity with the DataHub application.
```suggestion
# Origin of Energy shell
```

Reply:
Fixed